### ISSUE TITLE
Opening a new tab scrolls to the bottom, not the new tab

### ISSUE THREAD

**Issue by kumar303** [eoger/tabcenter-redux#163]
## To reproduce

* Install Tab Center Redux.
* Install [Always Right](https://addons.mozilla.org/en-us/firefox/addon/always-right/). This causes new tabs to open immediately after the current tab.
* Create enough new tabs so that your vertical tab list fills up the Firefox application window.
* Select a tab near the top of the list.
* Create a new tab.

## What happens:

The new tab is created immediately after the one you were on but the tab list scrolls to the bottom as if a new tab was created at the bottom.

## What should happen:

The tab list should not be scrolled to the bottom. Maybe there is some code that is scrolling to the bottom? Can it just select the newly created tab instead?

**Comment by Keith94** [comment]
The issue seems to be caused by limitations of the WebExtensions API, similar to: https://github.com/sblask/webextension-open-tabs-next-to-current/issues/26

**Comment by benmullins** [comment]
Always Right and Open Tabs Next to Current achieve the desired final tab position by moving the new tab to the position after the current tab. This means that the new tab is briefly at the end of the list, and Tab Center Redux scrolls to keep up with it. The issue is that Tab Center Redux doesn't scroll back; the native Firefox tab bar does do so, so the issue is less noticeable there than it is when using Tab Center Redux.

**Comment by kumar303** [comment]
> Tab Center Redux scrolls to keep up with it

Oh really? Do you or anyone know where in the code that happens?

**Comment by benmullins** [comment]
No. I'm just inferring this based on what I know about the behavior of the other extensions. Rather than just creating a new tab in place, the new tab is created in its normal position at the end of the list and then moved to its destination immediately after the previously open tab. Since after all of this, Tab Center Redux is scrolled all the way to the bottom, I'm just making an assumption that it happens at that point.

**Comment by kumar303** [comment]
If it's happening in Tabcenter Redux then it would be easy to fix! If it's happening in the WebExtensions tab API code then it would be harder to fix.

**Comment by Keith94** [comment]
Good news! The preference `browser.tabs.insertAfterCurrent` was finally [implemented](https://bugzilla.mozilla.org/show_bug.cgi?id=1344749).
It will open news tabs after the current tab without the scrolling issue described above. :)

**Comment by Keith94** [comment]
So, this will now be fixed for Firefox 61 and later.
